src/api-specification/auth-controller/authorizing-a-user-in-the-system.md

```diff
@@ -8,53 +10,86 @@
+}
+```
 
-For instructions on how to generate `signature` and `publicKey` strings, see [Generating key pairs](../../tutorials-api/generating-key-pairs.md) and [Signing user email addresses](../../tutorials-api/signing-user-email-addresses.md).
+:::
+
+::: tip Info
+
+For instructions on how to generate the `signature` string, see [Signing user email addresses](../../tutorials-api/signing-user-email-addresses.md).
+
+The `authPublicKeyHex` string is taken from the user's **Authorization** key pair on the FIB Web App **Profile** screen. For details, see [Web App UI: 'Authorization' key pair](../../overview/web-interface.md#akp).
 
 :::
 
-**Output/response structure**:
+:::
+
+#### Output/response structure
 
 ::: details Show
 
-```json5
+::: code-group Data structure
+
+```json5 [Structure]
+{
+  "status": {
+    "code": integer($int32),
+    "name": "string",
+    "message": "string"
+  },
+  "data": {
+    "accessToken": "string",
+    "refreshToken": "string",
+    "expirationTime": integer($int64)
+  }
+}
+```
+
+```json5 [Example]
 {
-  status: {
-    code: 0, //integer($int32)
-    name: 'string',
-    message: 'string'
+  "status": {
+      "code": 200,
+      "name": "OK",
+      "message": "User authorization has been successful."
   },
-  data: {
-    accessToken: 'string',
-    refreshToken: 'string',
-    expirationTime: 0 //integer($int64)
+  "data": {
+      "accessToken": "eyJ0eXAiOiJKV1QiLCJhbGciOiJFZERTQSJ9.eyJhY2MiOiJhbGljZUB3b25kZXJsYW5kIiwic3ViIjoiYWxpY2VAbWFpbC5jb20iLCJyb2xlIjoiUk9MRV9QRUVSIiwiZXhwIjoxNzAzMTg3NDEyLCJqdGkiOiIwNjgzM2FlZi03ZDljLTQ2MTUtYTU4YS02YjYxZWZkM2FhNjEifQ.HvrukWtcA7nz5rbxWiklJOjnv4X7jaAQ7piJaxqlW5rvUv52rClqYYHSn9F-P4u6pKNlJSpiiiH0GnynXsL7CQ",
+      "refreshToken": "eyJ0eXAiOiJKV1QiLCJhbGciOiJFZERTQSJ9.eyJzdWIiOiJhbGljZUBtYWlsLmNvbSIsImFjanRpIjoiMDY4MzNhZWYtN2Q5Yy00NjE1LWE1OGEtNmI2MWVmZDNhYTYxIiwiZXhwIjoxNzAzMjczNTEyLCJqdGkiOiJhY2NkYWUzNS00ODA1LTQ5NmQtYTBiZC02Y2Y4OGI3OTBkYjkifQ.rCrbFnF5axwixq4wwURZdu_DFwFlOil9RFYkmqlLYVPoU1UpN8PlWC8Kg7QBvA-eJYVzb8yKgQ0GiIAITe3ADA",
+      "expirationTime": 300
   }
 }
 ```
 
 :::
 
+:::
+
 ### Responses
 
-| Response Code | Description |
-| --- | --- |
-| `200` | User authorization has been successful. |
-| `401` | User signature is incorrect or failed to be decoded. |
-| `404` | User not found. |
-| `422` | User signature either could not be decoded or verified, or the `publicKey` string is not valid. |
-| `500` | Internal server error. |
+| Response Code | Description                                                                                     |
+| :-----------: | ----------------------------------------------------------------------------------------------- |
+| `200`         | User authorization has been successful.                                                         |
+| `401`         | User signature is incorrect or failed to be decoded.                                            |
```

Comment:
The request doesn't return a 401 error code, but it does return a 400 error code instead.